packages/gtk3.rb

```diff
@@ -59,16 +51,17 @@ def self.postinstall
     system "glib-compile-schemas #{CREW_PREFIX}/share/glib-2.0/schemas"
     # update mime database
     system "update-mime-database #{CREW_PREFIX}/share/mime"
-    puts
-    puts "To add basic settings, execute the following:".lightblue
-    puts "mkdir #{HOME}/.config/gtk-3.0".lightblue
-    puts "cat << 'EOF' > #{HOME}/.config/gtk-3.0/settings.ini
-[Settings]
-gtk-application-prefer-dark-theme = false
-gtk-icon-theme-name = hicolor
-gtk-fallback-icon-theme = gnome
-gtk-font-name = Arial 10
-EOF".lightblue
-    puts
+    
+    puts 'Performing configuration...'
+    FileUtils.mkdir_p("#{HOME}/.config/gtk-3.0")
```

Comment:
`#{CREW_DEST_HOME}` instead of `#{HOME}`?

Reply:
Actually, it would probably make sense to move this to the `def self.build` section instead.